Sources/protoc-gen-swift/Version.swift

```diff
@@ -17,6 +17,16 @@ struct Version {
     static let minor = 9
     static let revision = 28
     static let versionString = "\(major).\(minor).\(revision)"
+
+    // The "compatibility version" of the runtime library. This should be less
+    // than or equal to (major * 10000 + minor * 100 + revision). We guarantee
```

Comment:
That might help mitigate confusion about whether the version means "protos are **only** compatible with this version" rather than "protos are compatible with this version and up, until the next time we change it."

We could just use an integer starting at 1 that we increment every time we have to make a breaking change. WDYT?